lib/config.js

```diff
@@ -1,8 +1,7 @@
 // General
 
-exports.CONFIG_FILE = '.github/frictionless.yaml'
-exports.WORKING_DIRECTORY = '/github/workspace'
 exports.INQUIRY_PROPERTY = 'main'
+exports.CONFIG_FILE = '.github/frictionless.yaml'
```

Comment:
Hi @roll, in the ./github/frictionless.yaml I want to give a different input so I need to replace the variable CONFIG_FILE/ TABLE_GLOB?

main:
  tasks:
    - **path**: .csv

Reply:
Hi @naanrdk,

Can you please explaing in more details what you're trying to achieve?

Reply:
Use a custom file as input (s3 object) instead from the path data committed in the repository

Reply:
Then you just need to have `.github/frictionless.yaml`:

```yaml
main:
  tasks:
    - path: https://<path-to-s3-object>
 ```

Reply:
![image](https://user-images.githubusercontent.com/13908439/180713890-33c496ed-0c9e-4268-a2dc-c05cd695078b.png)

Reply:
@naanrdk 
It looks like a bug. Can you please create a bug report?

BTW are you sure there is no sensitive data you share or use in the `frictionless.yaml` . It's not intended to be used with secrets like AWS credentials because it might be stolen

Reply:
I understand it isn't ideal to send it publicly with key, I would like to use this actions in our pipeline to validate the data testing if frictionless repository would be a fit and find a way to send it secretly through environment variable. It would be nice if you could modularize to use it with any dataset by passing as a argument.

TIA

Reply:
Have you considered just using https://framework.frictionlessdata.io/ directly?

Reply:
Preferring to implement with repository over framework